easybuild/easyconfigs/m/mpi4py/mpi4py-2.0.0-intel-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'PythonPackage'
+
+name = 'mpi4py'
+version = '2.0.0'
+
+homepage = 'https://bitbucket.org/mpi4py/mpi4py'
+description = """MPI for Python (mpi4py) provides bindings of the Message Passing Interface (MPI) standard for
+ the Python programming language, allowing any Python program to exploit multiple processors."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+source_urls = [BITBUCKET_DOWNLOADS]
+sources = [SOURCE_TAR_GZ]
+
+py_maj_min = '2.7'
+pyver = '%s.12' % py_maj_min
+versionsuffix = '-Python-%s' % pyver
+
+dependencies = [('Python', pyver)]
```

Comment:
@jhein32 please update this to use the `%(pyver)s` template:

```python
name = 'mpi4py'
version = '2.0.0'
versionsuffix = '-Python-%(pyver)s'
...
dependencies = [('Python', '2.7.12')]
...
    'dirs': 'lib/python%(pyshortver)s/site-packages/mpi4py'],
```

you can add a new commit to this branch and push the updated branch to update this PR